antivir/models/quarantine.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+
+##############################################################################
+#
+#    Author: Nuncjo
```

Comment:
Please add yourself to Authors or create AUTHORS.rst file (for example https://github.com/psf/requests/blob/master/AUTHORS.rst)

Reply:
Thanks a lot.